app/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundService.scala

```diff
@@ -178,10 +178,10 @@ class OutboundService @Inject()(outboundConnector: OutboundConnector,
   }
 
   private def addOptionalAddressingHeaders(message: MessageRequest, wsaNs: OMNamespace, envelope: SOAPEnvelope): Unit = {
-    addWithAddressElementChildToSoapHeader(message.addressing.from, WSA_FROM, wsaNs, envelope)
+    addWithAddressElementChildToSoapHeader(x => x == x , message.addressing.from, WSA_FROM, wsaNs, envelope)
```

Comment:
This is clever but had me puzzled for a minute. Couldn't we just say `_ => true`?

Reply:
yes, it makes it much simpler. Done now